tests/resources/conftest.py

```diff
@@ -61,3 +65,54 @@ def headers():
         'content-type': 'application/json',
         'accept': 'application/json',
     }
+
+
+@pytest.fixture(scope="function")
+def resource_type_type(app):
+    """Resource type vocabulary type."""
+    return vocabulary_service.create_type(
+        system_identity, "resource_types", "rsrct")
+
+
+@pytest.fixture(scope="function")
```

Comment:
Had to make it function scoped and therefore remove it from `runnint_app`. The issue comes when used along `es_clear`. Since the vocabulary will be indexed and the labels come from ES, any previous test with `es_clear` will remove it and it will not be present when needed (e.g. labels). Note: this only happens at `tests/resources/` level since es_clear is not used otherwise.